python/oneflow/test/modules/test_consistent_empty.py

```diff
@@ -24,21 +24,43 @@
 from oneflow.test_utils.test_util import GenArgDict
 
 
-def _test_consistent_empty(test_case, shape, placement, sbp):
-    x = flow.empty(*shape, placement=placement, sbp=sbp)
+def _test_consistent_empty(test_case, func, shape, placement, sbp):
+    func2 = None
+    if func == "empty":
+        func = flow.empty
+    elif func == "new_empty":
+        func = flow.empty
+        func2 = flow.new_empty
+    else:
+        raise NotImplementedError
+
+    x = func(*shape, placement=placement, sbp=sbp)
+    if func2:
+        x = func2(x, size=shape)
 
     test_case.assertEqual(x.shape, flow.Size(shape))
     test_case.assertEqual(x.sbp, sbp)
     test_case.assertEqual(x.placement, placement)
 
 
-def _test_graph_empty(test_case, shape, placement, sbp):
+def _test_graph_empty(test_case, func, shape, placement, sbp):
+    func2 = None
+    if func == "empty":
+        func = flow.empty
+    elif func == "new_empty":
+        func = flow.empty
+        func2 = flow.new_empty
```

Comment:
```suggestion
    if func == "empty":
        func = flow.empty
    elif func == "new_empty":
        func = flow.empty
        func2 = flow.new_empty
```

这里用 `eval` 就可以省去 `if...elif` 的判断，是不是好点。 `eval` 就是把字符串当作代码执行，并且返回结果，比如：

```python
>>> import math
>>> func = eval("math." + "sin")
>>> func(5)
-0.9589242746631385
```

所以这里大概改成

```python
func = eval("flow." + func)
if func == "new_empty":
    func2 = flow.empty

就可以了（不过变量名什么的会需要根据实际情况调整）。 你看下要不要改一下。

Reply:
这块逻辑还没有那么一致，试着改了一下，反而会麻烦一些, 就不改了